_data/simple-icons.json

```
@@ -11171,6 +11171,11 @@
             "source": "https://brand.unity.com/",
             "guidelines": "https://brand.unity.com/"
         },
+        {
+            "title": "UnoCSS",
+            "hex": "000000",
+            "source": "https://github.com/unocss/unocss/blob/main/playground/public/icon.svg"
```

Please use a permalink to the logo file as described in the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md):

> In any case, the commit hash should be part of the URL. On GitHub, you can get the correct URL by pressing `y` on the GitHub page you want to link to. You can get help at the [getting permanent links to files page](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files).

```suggestion
            "source": "https://github.com/unocss/unocss/blob/fc2ed5bb6019b45565ff5293d4b650522f1b79b4/playground/public/icon.svg"
```